drivers/bridge/bridge_store.go

```diff
@@ -184,7 +184,7 @@ func (ncfg *networkConfiguration) Exists() bool {
 }
 
 func (ncfg *networkConfiguration) Skip() bool {
-	return ncfg.DefaultBridge
+	return false
```

Comment:
I changed it to false either. I made an assertion that if uses want to change default bridge network configs, he'll have to stop all the containers. So I still try to remove default bridge network on daemon start. But I gave a warning if I'm not able to do it.